raw_data/cpu_data/README.md

```diff
@@ -0,0 +1,27 @@
+Generating Data
+---------------
+
+AMD
+---
+
+Fileame:
+Download date: Jan 7, 2020
+Link: https://www.amd.com/en/products/specifications/processors
+
+Intel
+-----
+
+Fileame: intel_cpu_database.json
+Download date: Jan 7, 2020
+Link: https://github.com/divinity76/intel-cpu-database
+
+Verified data from ark.intel.com from the following repository
+
+Fileame: Intel_ARK_ComparisonChart_2020_01_07.csv
+Download date: Jan 7, 2020
+Link: https://ark.intel.com/content/www/us/en/ark/compare.html?productIds=194146,194145,195437,195434,192473,192472,192478,192471,192475,192470,192482,192481,192480,192477,192476,192474,192467,192465,193972,192451,193951,192447,192446,193969,192442,192448,193960,193959,192439,193955,193958,193954,193952,192450,193962,192445,193970,193950,193388,193953,193397,193564,193396,193392,193393,193391,193394,193389,193553,193385,193384,193382,193390,193381,193949,193957,192437,192444,192443,192440
```

Comment:
It would be nice if we could somehow get a list of all productIds

Reply:
Hundreds of results here.

https://ark.intel.com/content/www/us/en/ark/search/featurefilter.html?productType=873&2_StatusCodeText=4

Reply:
Even better:
https://ark.intel.com/libs/apps/intel/support/ark/advancedFilterSearch?productType=873&2_StatusCodeText=4&forwardPath=/content/www/us/en/ark/search/featurefilter.html&pageNo=25&sort=&sortType=

Edit pageNo to get 25 pages worth of productIDs

Reply:
Can we split this out as an issue?

Reply:
Sure, #188.